### ISSUE TITLE
Fix bug: Gantry 5 and k2 frontend editor system message

### ISSUE THREAD

**Issue by linkinformatica** [gantry/gantry5#1156]
When using the K2 frontend editor and after saving an article it doesn't show the system message "Article saved". So i fix it in ../media/gantry5/engines/nucleus/twig/component.html.twig adding "<jdoc:include type="message" />" .

Hope to help.

**Comment by Bokelmann** [comment]
... did you use the system-messages-particle?

http://docs.gantry.org/gantry5/particles/system-messages

If not try that.

**Comment by linkinformatica** [comment]
Yeah, in all my gantry Layout the system-messages-particle is present.

**Comment by mahagr** [comment]
There is System outline named Body Only, which by default contains both system messages and page content particles.

**Comment by mahagr** [comment]
@linkinformatica Any updates on your issue? Did you check the "Body Only" outline?

**Comment by linkinformatica** [comment]
@mahagr I tried, but it didn't work. As i wrote in the first post, i fix it by modifyng in ../media/gantry5/engines/nucleus/twig/component.html.twig adding the statement "jdoc:include type="message" " . 

I wanted to label this post as enhancement, but i didn't know how to do it.

**Comment by mahagr** [comment]
Only team members are able to change the labels.

Its very odd that you need to add `<jdoc>` for the message as system message particle does just that. 

What are the exact steps to reproduce the issue?

**Comment by linkinformatica** [comment]
I use joomla 3.4.8 with the latest version of Gantry package and Hydrogen template. As content management, i useK2 component and when i want to modify/add an article, i click save on the frontend pop editor and it doesn't show any system message after it refresh the editor.

**Comment by mahagr** [comment]
I'm not able to get K2 (v2.7.0) to work:

    Fatal error: Class 'K2HelperHTML' not found in /home/matias/www/joomla/administrator/components/com_k2/k2.php on line 80

**Comment by matthewc100** [comment]
Not sure it's resolved - I have this problem on fresh install for both 2.7.0 and 2.7.1.

**Comment by mahagr** [comment]
@matthewc100 Which template were you using? Also check how the `body only` layout looks like in your setup. It should have two green boxes in it; one for messages, another for content.
